R/trex.R

```diff
@@ -1,4 +1,39 @@
+#' Generate a Tableau Extension Manifest
+#'
+#' The `tableau_manifest()` function generates Tableau Extension Manifest XML
+#' which constitutes the extension manifest file (with the standard extension
+#' `.trex`). This XML text contains metadata for the extension and is used for
+#' registration.
```

Comment:
Perhaps make a note that most extension authors should look at yaml_skeleton and tableau_manifest_from_yaml instead?

Reply:
Added this in with a second paragraph of the function doc's description.